packages/alchemy/src/chains.ts

```diff
@@ -0,0 +1,50 @@
+import type { Chain } from "viem";
+import {
+  arbitrum,
+  arbitrumGoerli,
+  goerli,
+  mainnet,
+  optimism,
+  optimismGoerli,
+  polygon,
+  polygonMumbai,
+  sepolia,
+} from "viem/chains";
+import { GasFeeStrategy, type GasFeeMode } from "./middleware/gas-fees";
+
+export const SupportedChains = new Map<number, Chain>([
+  [polygonMumbai.id, polygonMumbai],
+  [polygon.id, polygon],
+  [mainnet.id, mainnet],
+  [sepolia.id, sepolia],
+  [goerli.id, goerli],
+  [arbitrumGoerli.id, arbitrumGoerli],
+  [arbitrum.id, arbitrum],
+  [optimism.id, optimism],
+  [optimismGoerli.id, optimismGoerli],
+]);
+
+const defineChainStrategy = (
+  chainId: number,
+  strategy: GasFeeStrategy,
+  value: GasFeeMode["value"]
+): [number, GasFeeMode] => {
+  return [chainId, { strategy, value }];
+};
+
+export const ChainFeeStrategies: Map<number, GasFeeMode> = new Map<
+  number,
+  GasFeeMode
+>([
+  // testnets
+  defineChainStrategy(goerli.id, GasFeeStrategy.FIXED, 0n),
+  defineChainStrategy(sepolia.id, GasFeeStrategy.FIXED, 0n),
+  defineChainStrategy(polygonMumbai.id, GasFeeStrategy.FIXED, 0n),
+  defineChainStrategy(optimismGoerli.id, GasFeeStrategy.FIXED, 0n),
+  defineChainStrategy(arbitrumGoerli.id, GasFeeStrategy.FIXED, 0n),
+  // mainnets
+  defineChainStrategy(mainnet.id, GasFeeStrategy.PRIORITY_FEE_PERCENTAGE, 25n),
+  defineChainStrategy(polygon.id, GasFeeStrategy.PRIORITY_FEE_PERCENTAGE, 25n),
+  defineChainStrategy(optimism.id, GasFeeStrategy.PRIORITY_FEE_PERCENTAGE, 5n),
```

Comment:
This should be `BASE_FEE_PERCENTAGE`

Reply:
Ah was going off of: https://github.com/OMGWINNING/rundler-infra/pull/32/files#diff-33c874a999d6039f3477b2d5511a90d4fcf9ac41b3548913cf802de517ecfafcR100

Reply:
Oh shoot, my bad, that's correctly configured in staging but not prod. `BASE_FEE_PERCENTAGE` is the correct value, I'll update that location.